src/wavcheck/read.py

```diff
@@ -112,7 +112,7 @@ def read_wav_files(ctx: Context) -> InternalState:
     result = InternalState()
     with os.scandir(ctx.dir) as entries:
         for entry in entries:
-            if entry.is_file() and str(entry.name).endswith(".wav"):
+            if entry.is_file() and (str(entry.name).endswith(".wav") or str(entry.name).endswith(".WAV")):
```

Comment:
What you have probably covers the only 2 likely cases, but how about `str(entry.name).lower().endswith(".wav"))` to make it slightly more general?

Reply:
@Lindurion I like it! Much more sensible 😄 Can you change before PR, or need me to reissue? Happy either way.